components/shopify_partner/sources/new_app_installs.js

```diff
@@ -0,0 +1,75 @@
+const shopify = require("../shopify_partner.app.js");
+const getAppInstalls = require("../queries/getAppInstalls");
+
+module.exports = {
+  key: "shopify_partner-new-app-installs",
+  name: "New App Installs",
+  type: "source",
+  version: "0.0.1",
+  description: "Emit new events when new shops install your app.",
+  props: {
+    shopify,
+    db: "$.service.db",
+    appId: {
+      type: "string",
+      optional: false,
+      description: "gid://partners/App/<your App ID here>",
+      label: "Shopify App ID",
+    },
+    occurredAtMin: {
```

Comment:
Since `appId`, `occurredAtMin` and `occurredAtMax` are defined twice, you can move those to the app file and then reference the `propDefinition` within each component. See https://pipedream.com/docs/components/api/#prop-definitions-example